web-components/src/components/toggle-switch/ToggleSwitch.ts

```diff
@@ -37,6 +41,20 @@ export namespace ToggleSwitch {
       };
     }
 
+    switchTemplate() {
+      if (this.alignLabel == "left") {
+        return html`
+          <slot></slot>
+          <span class="md-toggle-switch__label__container md-toggle-switch__label__container__left"></span>
```

Comment:
Can you add a part attribute on this label, so it can be targetted via CSS classes?
Part name to be: `toggle-label`